File: clients/typescript/src/AnonymousSchema_10.ts

export class AnonymousSchema_10 {
  private _files: (string | any)[];

  constructor(input: {
    files: (string | any)[],
  }) {
    this._files = input.files;
  }

  get files(): (string | any)[] { return this._files; }
  set files(files: (string | any)[]) { this._files = files; }
}

File: clients/rust/src/anonymous_schema10.rs
// AnonymousSchema10 represents a AnonymousSchema10 model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct AnonymousSchema10 {
    #[serde(rename="files")]
    files: Vec<crate::ReservedUnion>,
}

impl AnonymousSchema10 {
    pub fn new(files: Vec<crate::ReservedUnion>) -> AnonymousSchema10 {
        AnonymousSchema10 {
        files,
        }
    }
}
